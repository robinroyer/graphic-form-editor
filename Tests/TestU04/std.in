R r01 10 10 5 5
R r02 15 15 5 5
R r03 20 20 5 5
R r04 25 25 5 5
R r05 30 30 5 5
R r06 35 35 5 5
R r07 40 40 5 5
R r08 45 45 5 5
R r09 50 50 5 5
R r10 55 55 5 5
R r11 60 60 5 5
R r12 65 65 5 5
R r13 70 70 5 5
R r14 75 75 5 5
R r15 80 80 5 5
R r16 85 85 5 5
R r17 90 90 5 5
R r18 95 95 5 5
R r19 100 100 5 5
R r20 105 105 5 5
LIST
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
LIST
EXIT