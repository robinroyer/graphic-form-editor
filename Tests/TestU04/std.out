OK
#New object: r01
OK
#New object: r02
OK
#New object: r03
OK
#New object: r04
OK
#New object: r05
OK
#New object: r06
OK
#New object: r07
OK
#New object: r08
OK
#New object: r09
OK
#New object: r10
OK
#New object: r11
OK
#New object: r12
OK
#New object: r13
OK
#New object: r14
OK
#New object: r15
OK
#New object: r16
OK
#New object: r17
OK
#New object: r18
OK
#New object: r19
OK
#New object: r20
R r01 10 10 5 5
R r02 15 15 5 5
R r03 20 20 5 5
R r04 25 25 5 5
R r05 30 30 5 5
R r06 35 35 5 5
R r07 40 40 5 5
R r08 45 45 5 5
R r09 50 50 5 5
R r10 55 55 5 5
R r11 60 60 5 5
R r12 65 65 5 5
R r13 70 70 5 5
R r14 75 75 5 5
R r15 80 80 5 5
R r16 85 85 5 5
R r17 90 90 5 5
R r18 95 95 5 5
R r19 100 100 5 5
R r20 105 105 5 5
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'undo'ed
